agent/templates/agent/supervisor.conf.jinja2

```diff
@@ -31,5 +31,18 @@ stderr_logfile={{ directory }}/logs/worker.error.log
 user={{ user }}
 directory={{ directory }}
 
+{% if is_proxy_server %}
+[program:nginx_reload_manager]
+command=bash -c "{{ directory }}/repo/wait-for-it.sh redis://127.0.0.1:{{ redis_port }} && exec {{directory}}/env/bin/python {{ directory }}/repo/agent/nginx_reload_manager.py"
```

Comment:
Do signals propagate properly in this configuration?

Reply:
Yeah signal receiving is working fine.